build/webpack.config.dev.js

```diff
@@ -32,6 +34,10 @@ const webpackConfigDev = webpackMerge(webpackConfigBase, {
       filename: 'css/[name].css',
       chunkFilename: 'css/[id].css',
     }),
+    
+    new webpack.DefinePlugin({
+      'MAPBOXTOKEN': "'YOURMAPBOXTOKEN'"

```

Comment:
This is then replaced by the actual token when deploying ? If yes it would be good to add a comment in the readme about it and a comment in the code on line 37 to explicitly mention it